docs/guide.rst

```diff
@@ -2,4 +2,7 @@
 User Guide
```

Comment:
I think removing this page for now would be better. 

We probably won't have time to write a full guide. I would rather we focus on the remaining endpoints.

Reply:
Let's leave it like this for now. I want to add some basic info to it, separate from this PR.